developer_guide/installation/system_requirements/system_requirements.rst.inc

```diff
@@ -0,0 +1,130 @@
+
+**Operating Systems**
+
+Akeneo PIM application’s behaviour has been tested on some operating systems only. We cannot guarantee the behaviour of the application on servers different from:
```

Comment:
I would add that only Linux server are supported, and we tested on the following versions.